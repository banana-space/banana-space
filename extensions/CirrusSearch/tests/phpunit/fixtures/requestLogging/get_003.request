{
	"_comment": "Request multiple existing pages",
	"type": "get",
	"docIds": [25,26,27]
}
