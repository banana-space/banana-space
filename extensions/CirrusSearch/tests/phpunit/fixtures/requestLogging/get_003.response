[
    {
        "took": 2,
        "timed_out": false,
        "_shards": {
            "total": 8,
            "successful": 8,
            "failed": 0
        },
        "hits": {
            "total": 3,
            "max_score": 1,
            "hits": [
                {
                    "_index": "wiki_content_first",
                    "_type": "page",
                    "_id": "25",
                    "_score": 1,
                    "_source": {
                        "version": 27,
                        "wiki": "wiki",
                        "namespace": 0,
                        "namespace_text": "",
                        "title": "\u00c1frica",
                        "timestamp": "2016-09-20T22:44:04Z",
                        "category": [],
                        "external_link": [],
                        "outgoing_link": [],
                        "template": [],
                        "text": "for testing",
                        "source_text": "for testing",
                        "text_bytes": 11,
                        "language": "en",
                        "heading": [],
                        "opening_text": null,
                        "auxiliary_text": [],
                        "defaultsort": false,
                        "coordinates": [],
                        "redirect": [],
                        "incoming_links": 0
                    }
                },
                {
                    "_index": "wiki_content_first",
                    "_type": "page",
                    "_id": "26",
                    "_score": 1,
                    "_source": {
                        "version": 28,
                        "wiki": "wiki",
                        "namespace": 0,
                        "namespace_text": "",
                        "title": "10.1093\/acprof:oso\/9780195314250.003.0001",
                        "timestamp": "2016-09-20T22:44:27Z",
                        "category": [],
                        "external_link": [],
                        "outgoing_link": [],
                        "template": [],
                        "text": "10.1093\/acprof:oso\/9780195314250.003.0001",
                        "source_text": "10.1093\/acprof:oso\/9780195314250.003.0001",
                        "text_bytes": 41,
                        "language": "en",
                        "heading": [],
                        "opening_text": null,
                        "auxiliary_text": [],
                        "defaultsort": false,
                        "coordinates": [],
                        "redirect": [],
                        "incoming_links": 0
                    }
                },
                {
                    "_index": "wiki_content_first",
                    "_type": "page",
                    "_id": "27",
                    "_score": 1,
                    "_source": {
                        "version": 29,
                        "wiki": "wiki",
                        "namespace": 0,
                        "namespace_text": "",
                        "title": "10.5194\/os-8-1071-2012",
                        "timestamp": "2016-09-20T22:44:32Z",
                        "category": [],
                        "external_link": [],
                        "outgoing_link": [],
                        "template": [],
                        "text": "10.5194\/os-8-1071-2012",
                        "source_text": "10.5194\/os-8-1071-2012",
                        "text_bytes": 22,
                        "language": "en",
                        "heading": [],
                        "opening_text": null,
                        "auxiliary_text": [],
                        "defaultsort": false,
                        "coordinates": [],
                        "redirect": [],
                        "incoming_links": 0
                    }
                }
            ]
        }
    }
]
