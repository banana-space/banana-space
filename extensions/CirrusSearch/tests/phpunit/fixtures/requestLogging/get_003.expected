[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits": [
                        {
                            "index": "wiki_content_first",
                            "page_id": 25,
                            "page_title": "\u00c1frica",
                            "score": 1
                        },
                        {
                            "index": "wiki_content_first",
                            "page_id": 26,
                            "page_title": "10.1093\/acprof:oso\/9780195314250.003.0001",
                            "score": 1
                        },
                        {
                            "index": "wiki_content_first",
                            "page_id": 27,
                            "page_title": "10.5194\/os-8-1071-2012",
                            "score": 1
                        }
                    ],
                    "hits_offset": 0,
                    "hits_returned": 3,
                    "hits_total": 3,
                    "indices": [
                        "wiki"
                    ],
                    "max_score": 1,
                    "query_type": "get",
                    "search_time_ms": 2,
                    "suggestion_requested": false
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "docIds": [
                25,
                26,
                27
            ],
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 3,
            "hitsTotal": 3,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki",
            "indexType": false,
            "maxScore": 1,
            "queryType": "get",
            "source": "cli",
            "suggestionRequested": false,
            "tookMs": 0
        },
        "level": "debug",
        "message": "get of {indexType}.{docIds} against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]